problems/arrayMethods.js

```diff
@@ -45,7 +73,10 @@ const stringsOnly = () => {};
  * @returns {number[]} Only even valued elements
  */
 
-const onlyEvens = () => {};
+const onlyEvens = (nums) => {
+ let newArr = nums.filter(num => num %2 === 0)
```

Comment:
.filter returns an array so you can just put return in front of nums.filter instead of assigning it to a variable and returning the variable